tslint.json

```diff
@@ -5,7 +5,10 @@
     ],
     "linterOptions": {
         "exclude": [
-            "cypress/**"
+            "**/.!{ts,tsx}",
+            "node_modules/**",
+            "cypress/**",
+            "@Types/**"
```

Comment:
For VSCode. The script does not need this

Reply:
Isn't there a specific vscode config file you can dump this in?

Reply:
Not in this case. This is picked up by the extension